packages/cypress/src/support.ts

```diff
@@ -579,7 +579,7 @@ function register() {
           logs: () => [],
           add: () => {},
           get: (key?: any) => (!key ? log : log[key]),
-          toJSON: () => log,
+          toJSON: () => [],
```

Comment:
the previous version was completely crashing Cypress, the `toJSON`'s type here expects `string[]` so I used an empty array to ensure that it's compatible. In Cypress 10 `log` was typed as `any` and maybe at that time it worked OK, dunno. Today it's an object so we can see how it isn't compatible with the expected `string[]`.

I verified that we can still render `cy.log` just fine with this. It's quite plausible that we just don't rely on the return value of this function.